packages/compat/src/resolver-transform.ts

```diff
@@ -111,6 +111,16 @@ export function makeResolverTransform(resolver: Resolver) {
             }
           }
         },
+        ElementModifierStatement(node: ASTv1.ElementModifierStatement) {
+          if (node.path.type !== 'PathExpression') {
+            return;
+          }
+          if (node.path.this === true) {
```

Comment:
we need same for `node.path.data === true`  ({{@fooBar}})

Reply:
Thanks, didn't know about that!

Reply:
Added! I also forgot about "contextual" modifiers so I added that as well.

Reply:
@Windvis could we cover "modifier" helper usage?

like

```hbs
{{let (modifier "foo-bar" arg param=1) as |my-modifier|}}
   <div {{my-modifier}} ></div>
{{/let}}
```